utils/setNestedProp.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @param {any} parent
+ * @param {String} path
+ * @param {any} value
+ */
+export function setNestedProp(parent, path, value) {
```

Comment:
Let's export this from the module. It could be useful at blocks in the future, for deep template data for example.

Reply:
Yep.